stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/util/ChannelUtils.kt

```diff
@@ -28,7 +28,7 @@ public fun Channel.getDisplayName(): String {
         ?: getUsersExcludingCurrent()
             .joinToString { it.name }
             .takeIf { it.isNotEmpty() }
-        ?: stringResource(id = R.string.stream_compose_channel_list_untitled_channel)
+        ?: stringResource(id = R.string.stream_compose_untitled_channel)
 }
```

Comment:
Reused in the message list header. No need to duplicate the string as it is the presentation of an unnamed channel, regardless of the component (channel list, message list header, etc.)